src/Utils/Forcings/forcing_held_suarez.jl

```diff
@@ -0,0 +1,94 @@
+# Held-Suarez forcing
+struct HeldSuarezForcing{S} <: AbstractAtmosForcing
+    parameters::S
+end
+
+## this parameters should goes into runscript
+# model = Nonhydrostatic3DModel(......; source = (; forcing = HeldSuarezForcing(held_suarez_parameters)) )
+# day = FT(86400)
+# held_suarez_parameters = (;
+#     k_a = FT(1 / (40 * day)),
+#     k_f = FT(1 / day),
+#     k_s = FT(1 / (4 * day)),
+#     ΔT_y = FT(60),
+#     Δθ_z = FT(10),
+#     T_equator = FT(315),
+#     T_min = FT(200),
+#     σ_b = FT(7 / 10),
+#     R_d  = parameters.R_d,
+#     day  = parameters.day,
+#     grav = parameters.g,
+#     cp_d = parameters.cp_d,
+#     cv_d = parameters.cv_d,
+#     MSLP = parameters.p0,
+# )
+
+function rhs_forcing!(dY, Y, Ya, p, hs::HeldSuarezForcing, ::TotalEnergy)
+    # # unpack cache 
+    # TODO: make cache capability work for 3D model
+    # p = Ya.p
+
+    # unpack hs parameters
+    @unpack k_a, k_f, k_s, ΔT_y, Δθ_z, T_equator, T_min, σ_b = hs.parameters
+
+    R_d = CLIMAParameters.Planet.R_d(hs.parameters)
+    cp_d = CLIMAParameters.Planet.cp_d(hs.parameters)  
+    cv_d = CLIMAParameters.Planet.cv_d(hs.parameters)
+    p_0 = CLIMAParameters.Planet.MSLP(hs.parameters)
+
+    # obtain latitude
+    c_local_geometry = Fields.local_geometry_field(Y.base.ρ)
+    φ = deg2rad.(c_local_geometry.coordinates.lat),
+
+    # forcing
```

Comment:
The temperature relaxation that follows here should be put into a separate function, so it can be re-used for the different thermodynamic variables, rather than duplicating code.